docs/custom-flows/embedded-magic-links.mdx

```diff
@@ -5,7 +5,7 @@ description: Learn how to build custom magic link flows to increase user engagem
 
 # Embeddable magic links
 
-A "magic link" is a link, that when pressed, will automatically authenticate your user, so that they can quickly perform some action on your site with less friction than if they had to sign in manually.
+A ["magic link"](https://clerk.com/blog/magic-links) is a link, that when pressed, will automatically authenticate your user, so that they can quickly perform some action on your site with less friction than if they had to sign in manually.
```

Comment:
```suggestion
A ["magic link"](/blog/magic-links) is a link, that when pressed, will automatically authenticate your user, so that they can quickly perform some action on your site with less friction than if they had to sign in manually.
```

I believe a relative link here is fine 👍